tests/compat.sh

```diff
@@ -810,6 +800,10 @@ run_client() {
     LEN=$(( 72 - `echo "$TITLE" | wc -c` ))
     for i in `seq 1 $LEN`; do printf '.'; done; printf ' '
 
+    # Translate ciphersuite names based on client's naming convention
+    t_cipher=$(./scripts/translate_ciphers.py $3 $2)
```

Comment:
I don't understand why `run_client` has a third argument to indicate the client type: it seems redundant with `$1`.

Please either calculate the argument to pass to `translate_ciphers` from `name`, or document the new parameter in the comment above the function definition.

Reply:
I didn't realize we could calculate the argument from `$1` initially. Will change it later.